packages/venia-ui/lib/components/Header/cartCounter.js

```diff
@@ -1,23 +1,27 @@
-import React, { Component } from 'react';
-import classify from '../../classify';
-import PropTypes from 'prop-types';
+import React from 'react';
+import { number, shape, string } from 'prop-types';
 
+import { mergeClasses } from '../../classify';
 import defaultClasses from './cartCounter.css';
 
-class CartCounter extends Component {
-    static propTypes = {
-        classes: PropTypes.shape({
-            root: PropTypes.string
-        }),
-        counter: PropTypes.number.isRequired
-    };
+const CartCounter = props => {
+    const { numItems } = props;
 
-    render() {
-        const { counter, classes } = this.props;
-        return counter > 0 ? (
-            <span className={classes.root}>{counter}</span>
-        ) : null;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    // Don't show anything both when we don't have data to show, and when numItems is zero (0).
```

Comment:
It appears that we used to _always_ show either 0 or the number of items. The conditional where we created this component passed the `item_qty` or `0`. This changes that behavior so I just want to call that out to ensure it was intentional.

Reply:
I don't think that has changed, `cartTrigger` had a [conditional in its render function](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/Header/cartCounter.js#L17-L19).

It shouldn't have ever displayed a zero.

Reply:
Oh you're right! My bad :)